doc/user/builders-writing.xml

```diff
@@ -187,10 +115,10 @@ bld = Builder(action = 'foobuild &lt; $SOURCE &gt; $TARGET')
 
     <scons_example name="builderswriting_ex1">
        <file name="SConstruct">
-bld = Builder(action = 'foobuild &lt; $SOURCE &gt; $TARGET')
-env = Environment(BUILDERS = {'Foo' : bld})
 import os
-env['ENV']['PATH'] = env['ENV']['PATH'] + os.pathsep + os.getcwd()
+bld=Builder(action = 'foobuild &lt; $SOURCE &gt; $TARGET')
+env = Environment(BUILDERS={'Foo': bld})
+env.AppendENVPath('PATH', os.getcwd())
```

Comment:
Would Dir('.') work here? instead of os.getcwd()?

Reply:
Made this change, after checking it indeed works.

Reply:
Made that change a few places.